modules/blockexplorer/data.go

```diff
@@ -0,0 +1,66 @@
+package blockexplorer
+
+import (
+	"errors"
+
+	"github.com/NebulousLabs/Sia/build"
+	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/types"
+)
+
+// Calculates the total number of coins that have ever been created
+// from the bockheight
+func (be *BlockExplorer) totalCurrency() types.Currency {
```

Comment:
It's probably worth having this function accept a blockheight as input, and then not being of type 'BlockExplorer'